packages/components/src/components/navigation/navbar/navbar-profile.tsx

```diff
@@ -154,6 +155,7 @@ export class NavbarProfile {
               <slot  />
             </span>
           </div>
+          {this.userName && <div class='username__tooltip'>{this.userName}</div>}
```

Comment:
@tishoyanchev 
When userName is set to string with just white spaces like `userName = "       "`,  the black dash is still visible:
![image](https://github.com/Infineon/infineon-design-system-stencil/assets/97229205/fb12a8ea-b415-4735-b0aa-b14507c53119)

You can use this.userName.trim() to remove whitespaces.